server/handlers.go

```diff
@@ -389,6 +389,8 @@ func (s *Server) postHandler(w http.ResponseWriter, r *http.Request) {
 
 			filename = url.PathEscape(filename)
 			relativeURL, _ := url.Parse(path.Join(s.proxyPath, token, filename))
+			deleteURL, _ := url.Parse(path.Join(s.proxyPath, token, filename, metadata.DeletionToken))
+			w.Header().Set("X-Url-Delete-"+token, resolveURL(r, deleteURL, s.proxyPort))
```

Comment:
you cannot send headers after sending body content below

Reply:
I did not understand this. Do you mean to say that we cannot set headers after writing to `w` using `Fprintf`? If so, then how does it work to handle multiple files in the loop?

Reply:
you cannot mix http response body and http response headers.

all the the headers must be sent before the body.

you can collect in a slice all the `getURL(r, s.proxyPort).ResolveReference(relativeURL).String()` at https://github.com/dutchcoders/transfer.sh/pull/435/files#diff-961509ed3ff668f3172148150903514fb3394d6ff090d6db78d986875b13689fR394

and after you sent all the header, range over the slice and execute `write fmt.Fprintln(w, etc`